### PR TITLE
Update config to include max-background-flushes

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Update config to include max-background-flushes.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/4111
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Review by ran-huang — APPROVED** [review]

**Comment by lilin90** [comment]
@tabokie PTAL

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -638,6 +638,12 @@ rocksdb 相关的配置项。
 + 默认值：8
 + 最小值：1
 
+### `max-background-flushes`
+
++ memtable 后台 flush 任务的最大并发数。
```
Comment (tabokie) [inline tikv-configuration-file.md:643]: how about "RocksDB 用于刷写 memtable 的后台线程数"?

Comment (TomShawn) [inline tikv-configuration-file.md:643]: @tabokie PTAL, thanks
```suggestion
+ RocksDB 用于刷写 memtable 的最大后台线程数。
```

Comment (tabokie) [reply tikv-configuration-file.md:643]: Looks good!

**Pull request merged**